gradle-plugin/build.gradle

```diff
@@ -27,10 +28,9 @@ group = 'com.google.cloud.tools'
 sourceCompatibility = 1.8
 
 dependencies {
-  implementation platform('com.google.cloud.tools:dependencies-parent:1.4.3')
   implementation 'com.google.cloud.tools:dependencies:1.4.3'
-  implementation 'com.google.guava:guava'
-  implementation 'org.apache.maven.resolver:maven-resolver-api'
+  implementation 'com.google.guava:guava:29.0-jre'
+  implementation 'org.apache.maven.resolver:maven-resolver-api:1.4.2'
```

Comment:
The official "publish" plugin does not seem to support BOM (`platform`) when it generates pom.xml :(

https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1516